src/extends/roomposition.js

```diff
@@ -106,6 +106,23 @@ RoomPosition.prototype.getLink = function () {
   return this.__link
 }
 
+RoomPosition.prototype.getActiveLink = function () {
+  const link = this.getLink()
+  if (!link) {
+    return false
+  }
+
+  if (this.__linkActive) {
+    return link
+  }
+
+  if (typeof this.__linkActive === 'undefined') {
+    this.__linkActive = link.isActive()
```

Comment:
`isActive` is kind of expensive, it would be nice if this could be skipped at RCL8.